src/app/list-user/list-user.component.html

```diff
@@ -1,80 +1,167 @@
 <div class="content">
     <div class="security-title">Admin panel</div>
-    <div class="table-wrapper">
-      <div class="table-content">
-        <div class="filter-row">
-          <div class="left-row">
-            <div class="filter-dropdown">
-              <select class="security-filter-select" [(ngModel)]="position">
-                <option value="">All</option>
-                <option value="Employee">Employee</option>
-                <option value="Admin">Admin</option>
-              </select>
-            </div>
-            <div class="filter-input">
-              <input
-                class="security-filter-input"
-                placeholder="Search by email"
-                [(ngModel)]="email"
-              />
-            </div>
-            <div class="filter-input">
-              <input class="security-filter-input" placeholder="By name" [(ngModel)]="firstName" />
-            </div>
-            <div class="filter-input">
-              <input class="security-filter-input" placeholder="By surname" [(ngModel)]="lastName"/>
+
+    <div class="security-tabs">
+      <div
+        [ngClass]="selectedTab === 'users' ? 'tab tab-active' : 'tab'"
+        (click)="setSelectedTab('users')"
+      >
+        <div class="tab-content">Users</div>
+      </div>
+      <div
+        [ngClass]="selectedTab === 'permissions' ? 'tab tab-active' : 'tab'"
+        class="tab"
+        (click)="setSelectedTab('permissions')"
```

Comment:
Ne oboji se tab u drugu boju, popravi to. Potrebno je da to popravis i za user tab.